features/support/folders_helpers.rb

```diff
@@ -0,0 +1,3 @@
+def git_root_directory
+  output_of "git rev-parse --show-toplevel"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.